component/exporter.go

```diff
@@ -24,10 +24,7 @@ import (
 // ExporterConfig is the configuration of a component.Exporter. Specific extensions must implement
```

Comment:
```suggestion
// ExporterConfig is the configuration of a component.Exporter. Specific exporters must implement
```

Reply:
Done separately since not related to this PR. https://github.com/open-telemetry/opentelemetry-collector/pull/6551